src/default-theme/variables.css

```diff
@@ -93,6 +93,8 @@ all of MD as it is not optimized for dense, information rich UIs.
   --jp-code-padding: 5px;
   --jp-code-font-family: monospace;
 
+  --jp-notebook-scroll-padding: 100px;
+
```

Comment:
Hate to nit-pick, but this variable is in the "fonts" section. Should we make a "padding" (or something) section in that file?